### DISCUSSION TITLE
Multiple Indicators in a single panel

### DISCUSSION THREAD

**Discussion by bertaveira** [foxglove/community#525] [Q&A]
The indicator panel is super useful for checking live the status of systems. We have over 10 indicators but it gets very not aesthetic to have over 10 panels very small. 

Would be nice to be able to add multiple indicators to a single panel and have them be very close together and organised. This would however influence the problem mentioned in foxglove/community#498 . Maybe then a type of constant text label indicating what each indicator is should be added.

Sketch of the idea:
![foxglove_indicator](https://user-images.githubusercontent.com/9644729/178121067-64887d6b-a9c2-4d80-aeda-9d8ac0b1bdd4.jpg)

**Comment by jtbandes** [comment]
cc @2metres for input.

I think allowing multiple indicators per panel is a bit redundant since you can already place multiple panels – but maybe we need to improve the ergonomics of arranging multiple panels (e.g. in a column or row).

What would the labels provide that you don't already get by customizing the text?
